install_dependencies.sh

```diff
@@ -106,98 +104,56 @@ fi
 ./configure --with-libodb=../libodb-2.4.0 && make
 cd ..
 
-echo "installing libbz2"
-# This is commented till the bzip.org site is recovered.
-# wget http://www.bzip.org/1.0.6/bzip2-1.0.6.tar.gz -O ./libbz2.tar.gz
-# tar zxf ./libbz2.tar.gz
-# Till then we can trust ubuntu archives.
-wget http://archive.ubuntu.com/ubuntu/pool/main/b/bzip2/bzip2_1.0.6.orig.tar.bz2 -O ./libbz2.tar.bz2
-tar jxf ./libbz2.tar.bz2
-cd bzip2-1.0.6 && make
-cd ..
-
-echo "installing libz"
-wget http://prdownloads.sourceforge.net/libpng/zlib-1.2.11.tar.gz?download -O ./libz.tar.gz
-tar zxf ./libz.tar.gz
-cd zlib-1.2.11 && cmake . && make
-cd ..
-
-dependencies_dir_abs_path=`pwd`
-
-echo "installing openssl"
-mkdir openssl
-#using latest openssl which supports mac arm as well
-wget https://www.openssl.org/source/openssl-1.1.1w.tar.gz -O ./openssl-1.1.1w.tar.gz
-tar xzf ./openssl-1.1.1w.tar.gz
-cd openssl-1.1.1w
-LIBS="-lcrypto -ldl" \
-./config -fPIC no-shared no-threads \
-        --prefix=$dependencies_dir_abs_path/openssl \
-        --openssldir=$dependencies_dir_abs_path/openssl
-make && make install_sw
-cd ..
+if [[ "$OS_NAME" != "osx" ]]
+then
+  #linux 
+  echo "installing libbz2"
+  # This is commented till the bzip.org site is recovered.
+  # wget http://www.bzip.org/1.0.6/bzip2-1.0.6.tar.gz -O ./libbz2.tar.gz
+  # tar zxf ./libbz2.tar.gz
+  # Till then we can trust ubuntu archives.
```

Comment:
This comment was added [6 years ago](https://github.com/EBIvariation/vcf-validator/commit/74949d848c85475382f18d4743f1b295dfdc1dbd). I think we can remove it since it's unlikely that their wordpress website  would come back anytime soon.
```suggestion
```